Use grpc-bom (or include io.grpc:grpc-protobuf-lite)

At present, the bom is using all dependent grpc jars and not grpc-bom itself.
This seems to be the reason: https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/942
Is there any developments on that, probably the root cause is fixed and we can use grpc-bom again?

Ref: https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/102a28f/boms/cloud-oss-bom/pom.xml#L134-L199

The above list is missing `io.grpc:grpc-protobuf-lite` - So we have to import that specific dependency manually even if we use `libraries-bom`. So if adding the `grpc-bom` is not option, could the `grpc-protobuf-lite` be added to the list instead?

I don't think we use grpc-protobuf-lite. But looks like grpc-protobuf itself uses grpc-protobuf-lite: [here](https://github.com/grpc/grpc-java/blob/master/protobuf/src/main/java/io/grpc/protobuf/ProtoUtils.java#L57) and this causes a transitive dependency.

So, If I use `libraries-bom` without `grpc-protobuf-lite` explicitly, I get this error message (snipped): 
```
+-com.spotify:bom-integration-test:6-SNAPSHOT
  +-com.google.api.grpc:grpc-google-cloud-trace-v2:1.2.6                   
    +-io.grpc:grpc-protobuf-lite:1.27.2 (managed) <-- io.grpc:grpc-protobuf-lite:1.33.1
and                                                                            
+-com.spotify:bom-integration-test:6-SNAPSHOT                                                                                                                  
  +-com.google.api.grpc:grpc-google-cloud-translate-v3beta1:0.78.3          
    +-io.grpc:grpc-protobuf-lite:1.27.2 (managed) <-- io.grpc:grpc-protobuf-lite:1.32.2       
and                                                                            
+-com.spotify:bom-integration-test:6-SNAPSHOT
  +-com.google.cloud:google-cloud-translate:1.95.3                           
    +-io.grpc:grpc-protobuf-lite:1.27.2 (managed) <-- io.grpc:grpc-protobuf-lite:1.32.2
and                                                                            
+-com.spotify:bom-integration-test:6-SNAPSHOT    
  +-com.google.cloud:google-cloud-vision:1.100.6                             
    +-io.grpc:grpc-protobuf-lite:1.27.2 (managed) <-- io.grpc:grpc-protobuf-lite:1.33.1
and                                                                            
+-com.spotify:bom-integration-test:6-SNAPSHOT                                  
  +-com.google.api.grpc:grpc-google-cloud-vision-v1:1.100.6                  
    +-io.grpc:grpc-protobuf-lite:1.27.2 (managed) <-- io.grpc:grpc-protobuf-lite:1.33.1
and                                                                                                                                                            
+-com.spotify:bom-integration-test:6-SNAPSHOT                                  
  +-com.google.api.grpc:grpc-google-cloud-vision-v1p1beta1:0.83.6          
    +-io.grpc:grpc-protobuf-lite:1.27.2 (managed) <-- io.grpc:grpc-protobuf-lite:1.33.1
and                                                                            
```

If I avoid the library, by having the dependency as 'provided'  the pass the maven enforcer:
```
<dependency>
    <groupId>io.grpc</groupId>
    <artifactId>grpc-protobuf-lite</artifactId>
    <version>[1.27.2,)</version>
    <scope>provided</scope>
</dependency>
```


the maven enforcer is pases, but build fails:
```
java.lang.NoClassDefFoundError: io/grpc/protobuf/lite/ProtoLiteUtils
	at io.grpc.protobuf.ProtoUtils.marshaller(ProtoUtils.java:57)
	at com.google.bigtable.v2.BigtableGrpc.getReadRowsMethod(BigtableGrpc.java:73)
	at com.google.cloud.bigtable.grpc.BigtableSession.setupWatchdog(BigtableSession.java:364)
	at com.google.cloud.bigtable.grpc.BigtableSession.setupInterceptors(BigtableSession.java:324)
	at com.google.cloud.bigtable.grpc.BigtableSession.<init>(BigtableSession.java:262)
	at com.spotify.collection2.Main.create(Main.java:148)
	at com.spotify.apollo.environment.ApolloEnvironmentModule$ApolloEnvironmentImpl.initialize(ApolloEnvironmentModule.java:154)
	at com.spotify.apollo.standalone.StandaloneModule.lambda$bindAppInit$0(StandaloneModule.java:40)
	at com.spotify.apollo.standalone.StandaloneModule.requestHandler(StandaloneModule.java:46)
	at com.spotify.apollo.standalone.StandaloneService.boot(StandaloneService.java:208)
	at com.spotify.apollo.standalone.StandaloneService.boot(StandaloneService.java:170)
	at com.spotify.apollo.standalone.StandaloneService.boot(StandaloneService.java:165)
	at com.spotify.collection2.Main.main(Main.java:86)
Caused by: java.lang.ClassNotFoundException: io.grpc.protobuf.lite.ProtoLiteUtils
```

--- a/boms/cloud-oss-bom/pom.xml
+++ b/boms/cloud-oss-bom/pom.xml
@@ -184,6 +184,11 @@
       </dependency>
       <dependency>
         <groupId>io.grpc</groupId>
+        <artifactId>grpc-protobuf-lite</artifactId>
+        <version>${io.grpc.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.grpc</groupId>
         <artifactId>grpc-services</artifactId>
         <version>${io.grpc.version}</version>
       </dependency>
@@ -206,7 +211,7 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-      
+
       <dependency>
         <groupId>com.google.api</groupId>
         <artifactId>api-common</artifactId>
@@ -226,7 +231,7 @@
          <groupId>com.google.api</groupId>
          <artifactId>gax-httpjson</artifactId>
          <version>${gax.httpjson.version}</version>
-       </dependency>       
+       </dependency>
        <dependency>
          <groupId>com.google.auth</groupId>
          <artifactId>google-auth-library-bom</artifactId>

